src/main/java/org/truffleruby/Main.java

```diff
@@ -100,8 +100,8 @@ public static void main(String[] args) {
 
         if (config.getShouldRunInterpreter()) {
             final String filename = displayedFileName(config);
-            try (final Engine engine = createEngine(config, filename);
-                 final PolyglotContext polyglotContext = engine.newPolyglotContextBuilder().setArguments(RubyLanguage.ID, config.getArguments()).build();) {
+            try (Engine engine = createEngine(config, filename);
+                 PolyglotContext polyglotContext = engine.newPolyglotContextBuilder().setArguments(RubyLanguage.ID, config.getArguments()).build()) {
```

Comment:
It's less noise visually, and the `try` already shows we are declaring a new variable.